### PR TITLE
Update training/input data

### PR DESCRIPTION
**Pull request by dfsnow**
This PR fixes the missing sales situation discovered in https://github.com/ccao-data/data-architecture/pull/334. It:

- Ingests the (now available) missing sales and adds a marker to them (`sv_added_later`)
- Runs a very basic set of sales validation heuristics to flag the new sales
- Retrains the model using the updated sales sample
- Updates the Desk Review workbooks with:
  - An indicator (and conditional formatting) for sales that were previously missing from the review sheets
  - A column containing the previous initial model value
  - Columns showing the nominal and percent change from the old model value to the new model value

### CODE REVIEW

File: params.yaml [hunk 1]
```diff
@@ -395,4 +395,4 @@ ratio_study:
 # upload
 export:
   triad_code: "1"
-  run_id: "2024-02-16-silly-billy"
+  run_id: "2024-03-11-pensive-manasi"
```
Comment (dfsnow) [inline params.yaml:398]: This is the new final run ID w SHAPs.

File: renv.lock [hunk 2]
```diff
@@ -283,7 +283,7 @@
       "RemoteUsername": "ccao-data",
       "RemoteRepo": "ccao",
       "RemoteRef": "master",
-      "RemoteSha": "6bdb2f2cab811c3450d91726a784804bd8bca971",
+      "RemoteSha": "fe992f9bb6e75c97a0c8fa742106096adbabb642",
```
Comment (dfsnow) [inline renv.lock:286]: This is just updating the CCAO package to the latest version. It's tangential to this PR.

**Review by wrridgeway — APPROVED** [review]
Looks good to me. I didn't take a look at the rendered desk review sheets.

**Review by jeancochrane — APPROVED** [review]
Some serious duct tape in this one, but I think it's probably the best option if we really can't run another round of sales val.

The workbook looks good too, let's just make sure to correct the number formatting on the `Average YoY ∆ %` column in the `Neighborhood Breakout` sheet to a percentage with 1 decimal point before we share the workbooks:

https://github.com/ccao-data/model-condo-avm/blob/f5870ca88443a1ea5dfcc1bf7ce1be28327d3780/pipeline/07-export.R#L520-L529

**Pull request merged**